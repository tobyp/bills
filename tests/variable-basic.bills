#!./bills
%set x 1
A	B.	2*x
