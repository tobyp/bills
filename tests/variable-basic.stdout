todo: B pays 2.0 to A
